portfolio/src/main/webapp/script.js

```diff
@@ -38,55 +38,93 @@ function addRandomFact() {
 }
 
 /**
- * Adds the data from DataServlet using async/await (the return values are used directly), and converts it to a JSON.
+ * Deletes all the comments from the 'Comments' servlet.
+ */
+async function deleteDataUsingAsyncAwait(){
+  // Retrieve the data from '/comments' and delete the comments from the admin page
+  const response = await fetch('/comments', {
+    method: 'DELETE',
+  });
```

Comment:
Eventually we'll need to add error handling here. Can you add it? Or at least put a `TODO: handle errors`